src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LightSms;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class LightSmsTransport extends AbstractTransport
+{
+    protected const HOST = 'www.lightsms.com';
```

Comment:
```suggestion
    protected const HOST = 'lightsms.com';
```
If it does not work without the `www.` we should add it here:
```diff
        $endpoint = sprintf(
+              'https://www.%s/external/get/send.php?login=%s&signature=%s&phone=%s&text=%s&sender=%s&timestamp=%s',
-              'https://%s/external/get/send.php?login=%s&signature=%s&phone=%s&text=%s&sender=%s&timestamp=%s',
              $this->getEndpoint(),
```

Reply:
well, if you have your own host exposing the API, we cannot know whether it forces a `www.` subdomain

Reply:
it is not my API :) I am just use it :)

Reply:
You are right @stof but the implementation which is provided here should be checked against the real api.
So my question is, do we really  need the www. ? We have a bridge, can't remember which one it is, which does only work with the www. Prefix.

Reply:
I can check it tomorrow if it is very important www or without www. I used main URL from api inside their project

Reply:
I checked url with mobile device (balance validaton). Works with www and without https 😏 I will try to remove tomorrow and send without www 😏 and write back 😏

Reply:
I checked. We cannot use without WWW.
We get another strange error validation from their system
```
LightSmsTransport.php on line 126:
array:1 [▼
  "" => array:1 [▼
    "error" => 31
  ]
]
```

Reply:
I made additional validation according this case
```
Unable to send the SMS: Closing direction to the user
```

Reply:
Return back www. Tests fails in SF core code

```
1) Symfony\Component\Notifier\Bridge\LightSms\Tests\LightSmsTransportTest::testCanSetCustomHost
Failed asserting that 'lightsms://www.host.test/external/get/send.php?from=from' contains "://host.test".

/var/www/symfony/src/Symfony/Component/Notifier/Test/TransportTestCase.php:99

2) Symfony\Component\Notifier\Bridge\LightSms\Tests\LightSmsTransportTest::testCanSetCustomHostAndPort
Failed asserting that 'lightsms://www.host.test:42/external/get/send.php?from=from' contains "://host.test:42".

/var/www/symfony/src/Symfony/Component/Notifier/Test/TransportTestCase.php:121

```